### PR TITLE
Update new router

### PR DESCRIPTION
**Pull request by taijuten**

### CODE REVIEW

File: src/components/auth/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        identifier: null,
+        password: null
+    };
+
+    constructor(
+        private _logger: Logger,
+        private _authenticator: Authenticator,
+        private _router: Router
+    ) {
+        console.log('log on page c');
```
Comment (sam-cook) [inline src/components/auth/login/login.component.ts]: bruh

File: src/components/common/navbar/-authenticated/-authenticated.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+  <li><a [routerLink]="['']">Events</a></li>
```
Comment (sam-cook) [inline src/components/common/navbar/-authenticated/-authenticated.component.html]: empty link?

File: src/components/common/navbar/navbar.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+})
+export class NavbarComponent {
+  constructor() { }
+
+  isAuthenticated() {
+    return (tokenNotExpired());
+  }
+
+  logout() {
+  	// this._authenticator.logout();
```
Comment (sam-cook) [inline src/components/common/navbar/navbar.component.ts]: Logging out without the hassle of actually logging out? ;)

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+    // Other module loader config
+    tslint: {
+        emitErrors: false,
+        failOnHint: false
+    }
+};
+
+function getBanner() {
+  return 'This is a sample that shows how to add authentication to an Angular 2 (ng2) app by @auth0';
```
Comment (sam-cook) [inline webpack.config.js]: need this?

File: src/components/common/navbar/-authenticated/-authenticated.component.html (hunk 2 above)
Comment (taijuten) [inline src/components/common/navbar/-authenticated/-authenticated.component.html]: in the new router, this means navigate to root

**Pull request merged**